### ISSUE TITLE
Support helm charts

### ISSUE THREAD

**Issue by mhausenblas** [kubernauts/kploy#1]
As [pointed out by James Strachan](https://medium.com/@jstrachan/good-effort-michael-have-you-seen-helm-which-is-a-similar-idea-and-is-a-little-further-along-bacb39da2dd), helm is a great K8S package manager. Adding direct support for deploying its charts would be a great feature.

**Comment by technosophos** [comment]
Yeah, let's keep in touch about this. Let me know if you have any questions about Helm's formats, conventions, and structures. I think we're approaching some of the same questions.

**Comment by mhausenblas** [comment]
Cool, thanks @technosophos — I've now started some simple steps in this direction. Will need to play around a bit more with it to figure if this is the right way ;)
